authlib/integrations/httpx_client/oauth2_client.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, client_id=None, client_secret=None,
                  revocation_endpoint_auth_method=None,
                  scope=None, redirect_uri=None,
                  token=None, token_placement='header',
-                 update_token=None, **kwargs):
+                 update_token=None, leeway=60, **kwargs):
```

Comment:
why default to `60`?

Reply:
Because that's the default we've used at the lower-level function as well: https://github.com/lepture/authlib/blob/421732d50cf1f956aff4afa6a2a53f1da2b725f7/authlib/oauth2/rfc6749/wrappers.py#L13

Other than that, I don't have a strong opinion.